src/pysource/sensitivity.py

```diff
@@ -45,7 +45,10 @@ def grad_expr(gradm, u, v, model, w=None, freq=None, dft_sub=None, isic=False):
         eq_g = [Eq(gradm, gradm - expr, subdomain=model.grid.subdomains['nofsdomain'])]
         eq_g += freesurface(model, eq_g)
     else:
-        eq_g = [Eq(gradm, gradm - expr)]
+        if model.is_viscoacoustic:
+            eq_g = [Eq(gradm, gradm + expr)]
+        else:
+            eq_g = [Eq(gradm, gradm - expr)]
```

Comment:
why the sign difference?

Reply:
Just because I am using `u[0].dt * v[0].dt` to do the crosscorrelation.